mdpow/tests/test_solvation.py

```diff
@@ -52,7 +52,12 @@ def test_solvation_octanol(setup, ff):
 def test_solvation_cyclohexane(setup):
     solvation(setup, "cyclohexane")
 
-@pytest.mark.xfail("gromacs.release().startswith('2019')")
+@pytest.mark.xfail(gromacs.release.startswith('4')
+                   or gromacs.release.startswith('5')
+                   or gromacs.release.startswith('2016')
+                   or gromacs.release.startswith('2018'),
```

Comment:
2018 may work with mixed solvent.

Reply:
Let's see if it works.

Reply:
You were right, 2018 also passes the test.